From 8875c57a6c2450b499142bfffb83595a4a91cf1f Mon Sep 17 00:00:00 2001
From: hanzongcheng <hanzongcheng@huawei.com>
Date: Fri, 25 Aug 2023 10:36:11 +0800
Subject: [PATCH] disable setuptools_scm

* The dtc package comes from openEuler and does not contain
  git information, it will report an error when building with
  setuptools_scm. Disable it to fix this error.

Signed-off-by: hanzongcheng <hanzongcheng@huawei.com>

diff --git a/setup.py b/setup.py
index e82a832..18388ec 100755
--- a/setup.py
+++ b/setup.py
@@ -49,9 +49,6 @@ class build_py(_build_py):
 
 setup(
     name='libfdt',
-    use_scm_version={
-        "root": srcdir,
-    },
     cmdclass = {'build_py' : build_py},
     setup_requires = ['setuptools_scm'],
     author='Simon Glass',
-- 
2.34.1

